code/API_definitions/EdgeCloud_LcM.yaml

```diff
@@ -0,0 +1,648 @@
+        '503':
+          $ref: '#/components/responses/503'
+ 
+  /edge-cloud-platforms:
+    get:
+      tags: 
+        - Mobile Edge Cloud Platform
+      summary: Retrieve a list of the operator’s MECs and their status
+      description: List of the operator’s MECs and their status, ordering the results by location and filtering by status (active/inactive/unknown)
+      operationId: getMecPlatforms
+      parameters:
+        - name: regionId
+          description: Human readable name of the geografical zone of the MEC. Defined by the OP.
+          in: query
+          required: false
+          schema: 
+              $ref: '#/components/schemas/RegionId'
+        - name: status
+          in: query
+          required: false
+          schema: 
+              $ref: '#/components/schemas/Status'
+      responses:
+        '200':
+          description: Available MEC Platforms 
+          content:
+            application/json:
+              schema:
+                type: array
+                items:                  
+                  $ref: '#/components/schemas/MecDetails'
+                minItems: 1
+        '401':
+          $ref: '#/components/responses/401'
+        '403':
+          $ref: '#/components/responses/403'
+        '404':
+          $ref: '#/components/responses/404'
+        '500':
+          $ref: '#/components/responses/500'
+        '503':
+          $ref: '#/components/responses/503'
+ 
+components:
+  securitySchemes:
+    oAuth2ClientCredentials:
+      type: oauth2
+      flows: 
+        clientCredentials: 
+          tokenUrl: '/oauth2/token'
+          scopes:
+            nbi-mgmt: Access to the APIs
+  schemas:
+    AppId:
+      type: string
+      format: uuid
+      description: A globally unique identifier associated with the artifact. Originating OP generates this identifier when artifact is submitted over NBI.
+    AppInstanceId:
+      type: string
+      format: uuid
+      description: A globally unique identifier associated with a running instance of an application. Originating OP generates this identifier. 
+
+    AppProvisioning:
+      description: Attributes required to provision an application
+      type: object
+      properties:
+        artifactId:
+          $ref: '#/components/schemas/ArtifactId'
+        appName:
+          $ref: '#/components/schemas/AppName'
+        edgeResourceName:
+          $ref: '#/components/schemas/EdgeResourceName'
+        regionId:
+          $ref: '#/components/schemas/RegionId'
+      required:
+        - artifactId
+        - appName
+    AppName:
+      type: string
+      description: Name given by the application provider
+    ArtifactId:
+      type: string
+      format: uuid
+      description: A globally unique identifier assosiated with the artifact. Originating OP generates this identifier when artifact is submitted over NBI.
+ 
+    ArtifactDetails:
+      properties:
+        name:
+          type: string
```

Comment:
we may refine this in terms of valid values for the name including minLength and maxLength and patterns for valid chars

Reply:
`ArtifactDetails` no longer exists and all patterns have been removed for simplicity. Anyhow this specific details (patterns) can be included in the v1.0.0